src/utils/power_of_two.hh

```diff
@@ -0,0 +1,28 @@
+#ifndef POWER_OF_TWO_HH
+#define POWER_OF_TWO_HH
+
+#include "narrow.hh"
+#include <bit>
+#include <cassert>
```

Comment:
This code did not compile for me.
I had to add `#include <cstdint>`.

Reply:
Will add.